model/auth_model_identity.go

```diff
@@ -88,17 +90,34 @@ func createPasswordSalt() []byte {
 	return passwordSalt
 }
 
-func createVerifyCode() string {
-	verifyCode := make([]byte, 4)
-	_, err := rand.Read(verifyCode)
-	if err != nil {
-		panic(err)
+type VerifyCodeType int
+
+const (
+	VerifyCodeDefault VerifyCodeType = iota
+	VerifyCodeNumeric
+)
+
+func createVerifyCode(verifyCodeType VerifyCodeType) string {
+
+	if verifyCodeType == VerifyCodeNumeric {
+		max := big.NewInt(1000000) // 10^6
+		n, err := rand.Int(rand.Reader, max)
+		if err != nil {
+			panic(err)
+		}
+		return fmt.Sprintf("%06d", n.Int64())
```

Comment:
The use of big here seems unnecessary since there are 18 base 10 digits in 63 bits. I would just simplify this to 

```
code := rand.Int63n(1000000)
return fmt.Sprintf("%06d", code)
```